runtime/test-common/build.gradle.kts

```diff
@@ -32,10 +32,21 @@ configurations.all {
 }
 
 dependencies {
+  implementation(project(":polaris-core"))
+  implementation(libs.jakarta.ws.rs.api)
   implementation(enforcedPlatform(libs.quarkus.bom))
   implementation("io.quarkus:quarkus-junit5")
+
   implementation(platform(libs.testcontainers.bom))
   implementation("org.testcontainers:testcontainers")
   implementation("org.testcontainers:postgresql")
+
+  implementation(libs.testcontainers.keycloak) {
+    exclude(group = "org.keycloak", module = "keycloak-admin-client")
+  }
+  // Keycloak Admin Client brings RESTEasy Classic, which conflicts with Quarkus RESTEasy Reactive;
+  // it must not be present during Quarkus augmentation otherwise Quarkus tests won't start.
+  compileOnly(libs.keycloak.admin.client)
```

Comment:
I spent 5 hours today trying to find a better solution for this, but in the end, having both RESTEasy Classic and Reactive on the classpath gets blocked here:

https://github.com/quarkusio/quarkus/blob/072896e711640752ffc3176677591c6d263360e8/extensions/resteasy-reactive/rest-common/deployment/src/main/java/io/quarkus/resteasy/reactive/common/deployment/ResteasyReactiveCommonProcessor.java#L109-L110

Maybe a better solution would be to isolate the Keycloak container in a separate module, and shade RESTEasy Classic.

Reply:
Having a server runner that launches pre-built jars would help here, I believe... There was a discussion or PR about that somewhere, right?

Reply:
+1, shading feels too much.

Reply:
shading is too much for this case :) I'd go with a custom running... not ok "as is" for now